src/docs/environment/index.mdx

```diff
@@ -213,16 +257,25 @@ And after doing that, reload your shell:
 exec "$SHELL"
 ````
 
-Any time the `.envrc` configuration changes (including the first time) you will be prompted to run `direnv allow` before any
+Any time the `.envrc` configuration changes (including the first load) you will be prompted to run `direnv allow` before any
 of the configuration will run. This is for security purposes and you are encouraged to inspect the changes before running this command.
 
 ### Customize your development environment variables
 
 If you want to personalize your environment variables, you can do so by creating a `.env` file. This file is
 ignored by `git`, thus, you will not be able to leak it into one of your PRs.
 
-Two examples of variables that you can place there are `SENTRY_SILENCE_DIRENV_NOTICE` to make `direnv` output less noisy
-and `SENTRY_DEVENV_NO_REPORT=1` to prevent the development environment to report errors to Sentry.
+Some useful variables to set in `.env`:
+
+- `SENTRY_SILENCE_DIRENV_NOTICE`: Make `direnv` output less noisy.
+- `SENTRY_DEVENV_NO_REPORT`: Do not report development environment errors to Sentry.io.
+- `SENTRY_DIRENV_DEBUG`: Print debug messages to stdout.
+
+A good starter `.env`:
+```shell {filename: .env}
+SENTRY_SILENCE_DIRENV_NOTICE=1
+SENTRY_DEVENV_NO_REPORT=1
+````
```

Comment:
Not all of the values supported in .env are documented anywhere, so this should serve as a
starting point in documenting these settings. An example .env has also been added to
improve ease of use. Just create the file, copy-paste the example, and you're set!

Reply:
Maybe we can create a `.env.example` in Sentry and we slowly add all variables with their *default* values.
Or we even `cp .env.example .env` as part of the bootstrap script if the file does not exist.

What do you think?

Reply:
I like that idea! To build onto that, I'm thinking we could also omit the explanations for the variables by including descriptions for them in the `.env.example` itself, assuming it's possible to include comments there. If not, then I can keep this list.

Our suggested defaults appear to include all of the supported variables right now, so once a non-default variable is introduced we could begin exploring how we would like to document those in this repository. Thoughts?

Reply:
For optional aspects of the dev env, I would like to keep it directly via the CLI in Sentry. There's no easy way of keeping the develop docs in sync with what is happening in Sentry.